src/smt/smt_engine.cpp

```diff
@@ -4199,17 +4194,13 @@ void SmtEnginePrivate::processAssertions() {
 
   dumpAssertions("pre-static-learning", d_assertions);
```

Comment:
Argh, sorry. I should have seen this before. You can remove the `pre-static-learning` and `post-static-learning` dumps as well. Those are automatically supplied by the PreprocessingPass class.